frontend/packages/console-app/src/components/user-preferences/UserPreferencePage.scss

```diff
@@ -0,0 +1,16 @@
+.co-user-preference-page-content {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--pf-global--BackgroundColor--100);
+  height: 100%;
+  padding: 0 var(--pf-global--spacer--lg) var(--pf-global--spacer--lg) 0;
+  &__tab-content {
+    margin: var(--pf-global--spacer--lg) 0 0 var(--pf-global--spacer--3xl);
+    &__item {
+        margin-top: var(--pf-global--spacer--lg);
+    }
+  }
+  .pf-c-tabs__scroll-button {
+    display: none;
+  }
```

Comment:
Okay got why you want drop these buttons. I believe its an error that the buttons are rendered when `isVertical` prop is true, or?

Can you please add a comment and if you want also open an issue at https://github.com/patternfly/patternfly-react